notebooks/Model.ipynb

```diff
@@ -20,53 +20,53 @@
+       "API version 1.0 - NeomarilModelClient(url=\"http://localhost:7070/api\", Token=\"eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IlFnc0JWQ0I5WFc0V1YtSkVCVkJiZyJ9.eyJodHRwczovL25lb21hcmlsLmRhdGFyaXNrLm5ldC9uZW9tYXJpbC1ncm91cCI6ImRhdGFyaXNrIiwiaHR0cHM6Ly9uZW9tYXJpbC5kYXRhcmlzay5uZXQvZW1haWwiOiJuZW9tYXJpbC1jaUBkYXRhcmlzay5pbyIsImh0dHBzOi8vbmVvbWFyaWwuZGF0YXJpc2submV0L3RlbmFudCI6ImRhdGFyaXNrIiwiaHR0cHM6Ly9uZW9tYXJpbC5kYXRhcmlzay5uZXQvdGVuYW50LWFjdGl2ZSI6dHJ1ZSwiaHR0cHM6Ly9uZW9tYXJpbC5kYXRhcmlzay5uZXQvdXNlci1hY3RpdmUiOnRydWUsImh0dHBzOi8vbmVvbWFyaWwuZGF0YXJpc2submV0L3JvbGUiOiJtYXN0ZXIiLCJpc3MiOiJodHRwczovL2Rldi1tazNvN2xhenhsZTMwaHdxLnVzLmF1dGgwLmNvbS8iLCJzdWIiOiJhdXRoMHw2NTY0Y2M0NTlkYzAzODhlNDVlMDQzZTciLCJhdWQiOlsiaHR0cHM6Ly9kZXYtbWszbzdsYXp4bGUzMGh3cS51cy5hdXRoMC5jb20vYXBpL3YyLyIsImh0dHBzOi8vZGV2LW1rM283bGF6eGxlMzBod3EudXMuYXV0aDAuY29tL3VzZXJpbmZvIl0sImlhdCI6MTcyOTE5NDQ0MiwiZXhwIjoxNzI5MjA1MjQyLCJzY29wZSI6Im9wZW5pZCBwcm9maWxlIGVtYWlsIGFkZHJlc3MgcGhvbmUgcmVhZDpjdXJyZW50X3VzZXIgdXBkYXRlOmN1cnJlbnRfdXNlcl9tZXRhZGF0YSBkZWxldGU6Y3VycmVudF91c2VyX21ldGFkYXRhIGNyZWF0ZTpjdXJyZW50X3VzZXJfbWV0YWRhdGEgY3JlYXRlOmN1cnJlbnRfdXNlcl9kZXZpY2VfY3JlZGVudGlhbHMgZGVsZXRlOmN1cnJlbnRfdXNlcl9kZXZpY2VfY3JlZGVudGlhbHMgdXBkYXRlOmN1cnJlbnRfdXNlcl9pZGVudGl0aWVzIG9mZmxpbmVfYWNjZXNzIiwiZ3R5IjoicGFzc3dvcmQiLCJhenAiOiJrckI2TWxHdmQ4R0FJQ1l3WE93SVptTkMwWTlUYVBBNyJ9.lA_inD8lMJ5CjDaH2X6j5kLl3G78wA4dg-GGdmOd_WcQexeLWGvsLtV0RXG683w7BcZRRkyrgQdpE0S6EECq0jEuPhgagkcruqEqz7-837l7gNt-FouVGnpp9yLdmgazJ07nTBmqA37XSUdpVUrUp4OAqTK9mnP2TnYHRmP2fJQNhHWAwWZkEbj-4y23dsZAkdmSuZKJn6TJaRJgfMEp00LprLS902IQezodsJup_65uMq_TnAN3mN45J4QQnT6l5j1rh9b06h2dMCajeLaHgBdwJvF3szDQrOIygkOGOUhtZgicTytl3vVEIGBRPmFURMVOa7UpVSqxeNR7zeWRHw\")"
```

Comment:
Is this a valid token? It must be removed from this document if this is the case @henrique-lh

Reply:
I tested locally, so it won't be accessible to users

Reply:
I decided to clear the outputs

https://github.com/datarisk-io/mlops-neomaril-codex/pull/59/commits/af0ee157f4f7189835746a16c0c39768199e8443